pkgs/development/python-modules/yamlpath/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, hiera-eyaml
+, mock
+, pytest-console-scripts
+, pytest-cov
+, pytestCheckHook
+, pythonOlder
+, ruamel-yaml
+}:
+let
+  pinned-ruamel-yaml = ruamel-yaml.overridePythonAttrs (
+    old: rec {
+      pname = "ruamel.yaml";
+      version = "0.17.10";
+      src = fetchPypi {
+        inherit pname version;
+        sha256 = "EGvI1txqD/fJGWpHVwQyA29B1Va3eca05hgIX1fjnmc=";
+      };
+    }
+  );
+in
+buildPythonPackage rec {
+  pname = "yamlpath";
+  version = "3.6.3";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "wwkimball";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "4lLKMMsjVWbnfiaOzdBePOtOwPN8nui3Ux6e55YdGoo=";
+  };
+
+  propagatedBuildInputs = [ pinned-ruamel-yaml ];
+  checkInputs = [ hiera-eyaml mock pytest-console-scripts pytest-cov pytestCheckHook ];
```

```suggestion
  checkInputs = [ hiera-eyaml mock pytest-console-scripts pytestCheckHook ];
```
We don't care about coverage